packages/ingest/recursion.js

```diff
@@ -40,7 +28,7 @@ async function recurOnDirectory(fn, currentPath, segments, position) {
       if (['-', 0].includes(item.type)) {
         files.push(item);
       } else if (['d', 1].includes(item.type)) {
-        const nextDir = (currentPath === '' ? item.name : `${currentPath}/${item.name}`);
+        const nextDir = (currentPath === '/' ? `/${item.name}` : `${currentPath}/${item.name}`);
```

Comment:
If somebody knows a more elegant way of doing this I am all 👂 👂

Reply:
So this avoids there being a double `//` between the `currentPath` and `item.name` if the `currentPath` **is not** `/`?

Reply:
No it is specifically for fixing the case of listing `/`. If we list `/`, find `item`, and recur into it, we want to list `/item`, not `//item`. Any other `currentPath` will be fine, as it will just be `<path>/item`.

Reply:
A more elegant solution came to me: dcc4286a1